### DISCUSSION TITLE
I don't know what's error , I input anything and is error but no other tips

### DISCUSSION THREAD

**Discussion by CachePeng** [lshqqytiger/stable-diffusion-webui-directml#549] [Q&A]
![ai4](https://github.com/user-attachments/assets/362b886a-4e39-4378-a20f-212defdfc98c)

**Comment by CS1o** [comment]
Hey, try to load a different model.
Make sure your on Python 3.10.11 64bit
Then delete the venv folder and relaunch.
Also disable or whitelist any Browser adblocker for the Webui.

**Reply by CachePeng** [reply]
Thanks,I retry it and it normal run now. 
it on Python 3.10.6 now and I try to load new model and meet quest again , I restart my computer and it can solve.
But I don't know why...
